### PR TITLE
Fixing `dragConstraints={ref}` with active layout animations

### PR DESCRIPTION
**Pull request by mattgperry**
Fixes https://github.com/framer/company/issues/29798

Ensures that draggable components are always remeasured without their transform.

### CODE REVIEW

File: packages/framer-motion/cypress/integration/drag.ts [hunk 1]
```diff
@@ -234,9 +234,9 @@ describe("Drag", () => {
             .get("[data-testid='draggable']")
             .trigger("pointerdown", 10, 10)
             .trigger("pointermove", 15, 15)
-            .wait(50)
+            .wait(200)
             .trigger("pointermove", 300, 300, { force: true })
-            .wait(50)
+            .wait(200)
```
Comment (nvh) [inline packages/framer-motion/cypress/integration/drag.ts:239]: Why is this required?

**Review by nvh — APPROVED** [review]

File: packages/framer-motion/cypress/integration/drag.ts (hunk 1 above)
Comment (mattgperry) [reply packages/framer-motion/cypress/integration/drag.ts:239]: The layout animation I added is like 100 - 200ms long so this delay has to be long enough for them to trigger and fuck up the measurements
